Source/Statuses/OktaAuthStatusFactorEnrollActivate.swift

```diff
@@ -61,16 +61,6 @@ open class OktaAuthStatusFactorEnrollActivate : OktaAuthStatus {
         return true
     }
 
-    open func activateFactor(passCode: String?,
```

Comment:
Please don't remove:
1. it is breaking change
2. dev can use `OktaAuthStatus` classes only without touching `OktaFactor`

Reply:
Done.